synedrion/tests/threshold.rs

```diff
@@ -179,7 +190,9 @@ fn full_sequence() {
         })
         .collect();
 
-    info!("\nRunning InteractiveSigning\n");
+    info!("AuxGen took {:?}", now.elapsed()); // ~42 sec
```

Comment:
42s applies to ProductionParams, not TestParams, I would hope?

Reply:
Yes! Sorry, I'll remove them. But yeah, it's slow af.